### PR TITLE
feature/1686 - Add RJAA (Tokyo Narita)

### LINKED ISSUE #1686: Add new airport RJAA (Tokyo Narita)
This is to add RJAA to the simulation.

- [x] Airspace
- [x] Airways
- [x] Base Airport Properties/Runways
- [x] SIDs
- [x] STARs
- [x] SpawnPatterns
- [x] Terrain
- [x] Video Map

### PR DESCRIPTION
**Pull request by JPNite3**
Resolves #1686 

The purpose of this pull request is to add Tokyo Narita (RJAA)

### CODE REVIEW

File: assets/airports/airportLoadList.json [hunk 1]
```diff
@@ -432,7 +432,6 @@
         "level": "hard",
         "name": "Tokyo Narita International Airport",
         "premium": false,
```
Comment (cake-pie) [inline assets/airports/airportLoadList.json:434]: Comma needs to be removed

Comment (JPNite3) [reply assets/airports/airportLoadList.json:434]: Yeah did realize that, fixed already. Thanks for the nudge!

**Comment by erikquinn** [comment]
@cake-pie Looks like this is still in progress... when it's ready for review, please re-request from airport-reviewers.

**Comment by JPNite3** [comment]
> @JPNite3  Looks like this is still in progress... when it's ready for review, please re-request from airport-reviewers.

Erm no, it's done...

**Comment by erikquinn** [comment]
@JPNite3 well you requested a review a while ago, but then did some more work here and there, and the PR has a `WIP` (work in progress) label on it....

So if it's done, I will do the following for you:
- remove the WIP label
- mark the PR as ready
- request a review from the airport-reviewers team

This is what you need to do if you want anything to be done with your PRs, otherwise they will just sit there while we wait for you to tell us it's ready.

**Comment by JPNite3** [comment]
> @JPNite3 well you requested a review a while ago, but then did some more work here and there, and the PR has a `WIP` (work in progress) label on it....
> 
> 
> 
> So if it's done, I will do the following for you:
> 
> - remove the WIP label
> 
> - mark the PR as ready
> 
> - request a review from the airport-reviewers team
> 
> 
> 
> This is what you need to do if you want anything to be done with your PRs, otherwise they will just sit there while we wait for you to tell us it's ready.

I'm so sorry. Did not realize at all despite it being my third airport! Thanks for letting me know and doing that.

**Review by Fechulo — APPROVED** [review]
Nice work! Sorry for the delay

File: assets/airports/rjaa.json [hunk 2]
```diff
@@ -1,1158 +1,2057 @@
...
-                ["ups",  4]
+				["amu", 14],
+				["ana", 57],
+				["apj", 56],
+				["cal", 35],
+				["cpa", 72],
+				["crk", 12],
+				["csz", 1],
+				["eva", 28],
+				["hke", 30],
```
Comment (erikquinn) [inline assets/airports/rjaa.json]: This airline does not currently exist, and is causing an error. Please add the airline to this PR.

File: assets/airports/rjaa.json [hunk 3]
```diff
@@ -1,1158 +1,2057 @@
...
+				["aic", 2],
+				["ana", 84],
+				["cca", 18],
+				["ces", 17],
+				["cqh", 7],
+				["csc", 1],
+				["csn", 9],
+				["cxa", 5],
+				["dkh", 7],
+				["edt", 7],
```
Comment (erikquinn) [inline assets/airports/rjaa.json]: This airline does not currently exist, and is causing an error. Please add the airline to this PR.

**Review by erikquinn — CHANGES_REQUESTED** [review]
Please add HKE/EDT airlines to clear errors.

**Review by erikquinn — APPROVED** [review]

**Pull request merged**